Akilahinternees/Andelachallenge
# bg(teamwork): modifying interface for teamwork project

> #### What does this PR do?
> Modify user interface of the project
>
> #### Description of Task to be completed
> I have changed some colors in different pages
>
> #### How should this be manually tested?
>By viewing the codes or run them to see if it meet expectations
> #### Any background context you want to provide?
>N/A
> #### What are the relevant pivotal tracker stories?
>ft-modify-interface-168781427
> #### Screenshots (if appropriate)
> N/A
